runner/src/main/java/io/github/rdfc/Reader.java

```diff
@@ -75,6 +89,51 @@ void close() {
         this.streams.forEach(stream -> stream.end());
```

Comment:
minor(style): it just looks nicer but you can ignore
```suggestion
        this.strings.forEach(StreamIter::end);
        this.streams.forEach(StreamIter::end);
```

Reply:
OOO looks good! I actually forgot this syntax existed